test/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../.eslintrc.json",
+  "env": {
```

Comment:
Alternately, this file could be deleted, and the config hoisted to an `overrides` section in the root config like https://eslint.org/docs/user-guide/configuring#specifying-processor